sys/fuchsia/hypervisor_guests.txt

```diff
@@ -0,0 +1,20 @@
+include <zircon/syscalls.h>
+include <ddk/driver.h>
+
+# Provided by sysroot (include/ddk/driver.h)
+resource zx_root_resource[zx_handle]
+get_root_resource() zx_root_resource
```

Comment:
Aha, so this is how one gets resources!

I think we need to move this to `sys.txt` as it's unrelated to hypervisor. And then we also need to address these TODO's (these need this resource too?):

```
sys/fuchsia/log.txt:# TODO: do we need root resource for this? kernel actually has debuglog_create syscall that accepts resource...
sys/fuchsia/objects.txt:# TODO: what is handle type here?
sys/fuchsia/objects.txt:# TODO: handle is "root resource", where does one get one?
sys/fuchsia/objects.txt:# TODO: what is handle type here?
```

And also it's probably time to reevaluate:
```
# TODO: temporary disabled as it crashes kernel left and right.
# zx_object_get_child(handle zx_handle, koid koid, rights flags[zx_rights], out ptr[out, zx_handle])
```

Reply:
Updated all the ones you mentioned, except for zx_object_get_child.  Plan to get to that one in a future PR, for the sake of keeping this one manageably sized + going home this weekend~

Reply:
Makes sense. I am merging this now. We can do the rest separately.
Thanks